setup.sh

```diff
@@ -108,26 +131,26 @@ function confirmChoices() {
 
 function installAnew() {
     local projects
-    projects=($*)
-    for project in $projects; do
+    projects=("$@")
+    for project in "${projects[@]}" ; do
         (
-            cd $wd &&
-            git clone --quiet git://github.com/dlang/$project.git &&
-            touch $tempdir/$project
+        git clone "git://github.com/$githubUser/$project.git" "$wd/$project"
+        if [ "$githubUser" != "dlang" ] ; then
+            git remote add upstream "git://github.com/dlang/$project.git"
+        fi
```

Comment:
Most pages in the wiki expect `upstream` to be set (they do explain how to do it, but there's no reason why we can't be nice here).